core/io/resource_loader.h

```diff
@@ -194,7 +194,6 @@ class ResourceLoader {
 	static ResourceLoaderImport import;
 
 	static bool add_custom_resource_format_loader(String script_path);
-	static void remove_custom_resource_format_loader(String script_path);
```

Comment:
Is this because resource loaders are cleaned up automatically? I don't see a strong reason to remove this, because it complements corresponding `add_` methods.

Reply:
Dunno, I've actually seen many methods that are added just to complement something, but then they are never used (usually it's just not necessary or it's done in a different way). Like, what is the use for `remove_custom_resource_format_loader()`? I mean, it removes a single loader. There's already `remove_custom_loaders()` methods that removes them all and can be useful for cleanup or something, but removing single loader? It would be a different thing if it was exposed for EditorPlugins, but it's C++-only code.

> This may be used in modules, so breaks compatibility.

I think we should comment every method intended to be used with modules (and I mean methods that are actually used by someone). It's hard to tell what can be removed tbh.